docker/apache2/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM webdevops/apache:ubuntu-18.04
```

Comment:
@mwarzybok-sumoheavy I'd rather not use `webdevops`, but instead stay with official images.

Reply:
@bobbrodie updated